options.go

```diff
@@ -0,0 +1,78 @@
+package osrm
+
+import (
+	"fmt"
+	"net/url"
+	"sort"
+	"strconv"
+)
+
+// options represents OSRM query params to be encoded in URL
+type options map[string][]string
+
+// Set saves a string value by the key
+func (opts options) Set(k, v string) {
```

Comment:
I think we can make all function unexported for `options` type.

Reply:
updated

Reply:
updated